### DISCUSSION TITLE
Breaking Changes

### DISCUSSION THREAD

**Discussion by GlennGoddard** [briis/hass-weatherflow2mqtt#115] [Ideas]
It would be nice if we document the breaking changes in one location.  I initially thought about the readme or a separate file with breaking changes listed, but then thought that maybe the discussion section is a good place; but either way we need it somewhere.

The latest breaking change (dev for now) is the separation of stations from the WF sensor.  I knew this was coming, so it was not hard to figure it out but we should start documenting these especially to make it easier for those using the stable branch.

**Comment by briis** [comment]
You are absolutely right Glenn. It was my mistake. I had too much stuff to do, so I did not take enough time to write that up properly. 
It must go in the CHANGELOG.md, which are the release notes.

**Comment by GlennGoddard** [comment]
I think we should call that out in the ReadMe for people to check the CHANGELOG.md for any breaking changes.  Although most are like me probably and update then just see what breaks.

**Reply by briis** [reply]
Agree, I will do that.
And btw. Nathan did update the Changelog a few hours ago, with a better description of the breaking changes and how to work around it.

**Comment by GlennGoddard** [comment]
Well...I guess I should look there now.
We need to reword the readme section on the weather card since it is not sensor.wf...... anymore.  The forecast would be the hub plus the serial number for me, and I assume it is the same for all others.  The other properties should be sensor.'tempest/air or sky' plus serial (this is a guess since I don't have access to anything but a tempest.)

**Reply by briis** [reply]
Yes. If you want to do the overhaul of the README, I would appreciate it.

**Comment by GlennGoddard** [comment]
Made simple addition to ReadMe for breaking changes.   The notes in ChangeLog where referenced which should be sufficient for the forecast section.

**Comment by briis** [comment]
Thanks Glenn. I merged this.

**Comment by GlennGoddard** [comment]
Sure, I will take credit for not much work.
